teams/relnotes-interest-group.toml

```diff
@@ -0,0 +1,12 @@
+# Intended to be a ping group on rust-lang/rust for release notes for people who
+# would like to be notified of new relnotes PR and want to double-check.
+
+name = "relnotes-interest-group"
+kind = "marker-team"
+
+[people]
+leads = []
+members = [
+    "jieyouxu",
+    "traviscross",
```

Comment:
cc @traviscross as you said you would like to get cc'd. Didn't add all of T-lang, other T-lang members can add themselves if wanted.